packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -641,6 +623,36 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
         require(_liveness > 0, "Liveness cannot be 0");
     }
 
+    function _getState(
```

Comment:
[related](https://github.com/UMAprotocol/protocol/pull/3188/files#r664719002), needed to make an internal method for other contract functions to fetch state from, while keeping interface the same.